src/qt/bitcoingui.cpp

```diff
@@ -104,7 +108,7 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *platformStyle, const NetworkStyle *n
 {
     GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
 
-    QString windowTitle = tr("Bitcoin Core") + " - ";
+    QString windowTitle = tr(PACKAGE_NAME) + " - ";
```

Comment:
This won't work. And you *don't* want the `tr`. Just use:

    QString windowTitle = QString(PACKAGE_NAME) + " - ";